### PR TITLE
PR from development to master branch

### PR DESCRIPTION
**Pull request by jfrogsolutioncicd**

### CODE REVIEW

File: TFproviderTest.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+            #- export ARTIFACTORY_URL=${jdp_url} && echo ${ARTIFACTORY_URL}
+            - export ARTIFACTORY_URL=https://tf-provider-rt-xray-1631848661-rt.jfrog.tech
+            - export ARTIFACTORY_USERNAME=admin
+            - export ARTIFACTORY_PASSWORD=${NEW_RT_PASSWORD} && echo ${ARTIFACTORY_PASSWORD}
+            - echo "Get cookie to generate Access token"
+            - >-
+              export COOKIES=$(curl -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
+                            --header "accept: application/json, text/plain, */*" \
+                            --header "content-type: application/json;charset=UTF-8" \
+                            --header "x-requested-with: XMLHttpRequest" \
```
Comment (chb0github) [inline TFproviderTest.yml:186]: This for sure isn't needed.

File: TFproviderTest.yml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+                            -d '{"user":"admin","password":"'"${ARTIFACTORY_PASSWORD}"'","type":"login"}' | grep FALSE)
+            - echo "${COOKIES}"
+            - export REFRESHTOKEN=$(echo $COOKIES | grep REFRESHTOKEN | awk '{print $7 }')
+            - export ACCESSTOKEN=$(echo $COOKIES | grep ACCESSTOKEN | awk '{print $14 }')
+            - >-
+              export ACCESS_KEY=$(curl -g --request GET "${ARTIFACTORY_URL}/ui/api/v1/system/security/token?services[]=all" \
+                                  --header "accept: application/json, text/plain, */*" \
+                                  --header "x-requested-with: XMLHttpRequest" \
+                                  --header "cookie: ACCESSTOKEN=${ACCESSTOKEN}; REFRESHTOKEN=${REFRESHTOKEN}")
+            - echo ${REFRESHTOKEN}
```
Comment (chb0github) [inline TFproviderTest.yml:196]: remove these echos

File: TFproviderTest.yml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+            - echo ${ACCESSTOKEN}
+            - echo ${ACCESS_KEY}
+            - add_run_variables ARTIFACTORY_ACCESS_TOKEN=${ACCESS_KEY}
+            - export TF_ACC=1
+            #- make acceptance # --- TODO: RT test TestAccPermissionTarget_full fails
```
Comment (chb0github) [inline TFproviderTest.yml:201]: Like I said, you can comment out that line

**Review by chb0github — CHANGES_REQUESTED** [review]
A few things. Most importantly, don't echo those security credentials

**Comment by danielmkn** [comment]
Closing this one, will open a new one

**Pull request closed without merging**